demystifying_large_language_models/demystifying_large_language_models.md

```diff
@@ -0,0 +1,691 @@
+
+6. Automation, Access, and Environmental Harms
+---
+
+LLMs require significant energy (with its associated carbon burden) as well as other resources, such as water for data center cooling needs and materials for computer chips.  The extraction and use of these resources can harm the environment and directly and indirectly harm people.  People are also potentially harmed by other aspects of LLMs, including employment related risks as jobs are automated or made less creative by the use of LLMs.  Artists and authors, for example, are already contending with threats to their livelihood and the use of their works as training data.  Finally, there is a risk of widening the digital divide and exacerbating differences in access to the benefits of technology.
+
+<div class = "important">
+<b style="color: rgb(var(--color-highlight));">Important note</b><br>
+
+![A figure with a ponytail indicates some words projected onto a screen while saying "An analysis of our new AI hiring algorithm has raised some concerns."  On the screen is displayed "DeepAIHire Candidate Evaluation Algorithm, Inferred Internal Weightings.  What follows is a table of weights that each apply to various factors.  There are four tiny weights and one enormous weight.  They are: weight 0.0096 for the factor of Educational Background, weight 0.520 for the factor of Past Experience, 0.0208 for Recommendations, 0.0105 for Interview Performance and a huge 783.5629 for the factor described as "Enthusiasm for developing and expanding the use of the DeepAIHire Algorithm."](https://imgs.xkcd.com/comics/ai_hiring_algorithm_2x.png "[AI Hiring Algorithm](https://xkcd.com/2237/) comic by xkcd, [CC BY-NC 2.5](https://xkcd.com/license.html).")<!-- style = "max-width: 280px;" -->This risk framework reflects a point in time, and you may think of risk types or examples that aren't reflected here. Suffice it to say, LLMs, like any laboratory tool, must be used with discretion and an awareness of the risks, costs, tradeoffs, and utility they provide.
+
+</div>
+
+## Benefits of LLMs
+
+Any process that uses language can potentially benefit from LLM technology.  Consider some of these use cases, which we supplement with examples of prompts we supplied and replies we received from ChatGPT:
```

Comment:
Consider revamping this page / the prompts to produce much shorter examples? If the intention is for folks to actually engage with the examples, these are so long I don't think anyone will read more than a line or two. This module is already really text heavy, and this page could contribute to a sense of overwhelm by how dense the module is (Even if its not expected for folks to carefully read everything)